sw/device/lib/dif/dif_aes_unittest.cc

```diff
@@ -39,14 +39,17 @@ class AesTest : public testing::Test, public mock_mmio::MmioTest {
       EXPECT_WRITE32(offset, iv.iv[i]);
     }
   }
-  void setExpectedConfig(uint32_t key_len, uint32_t mode, uint32_t operation) {
+  void setExpectedConfig(uint32_t key_len, uint32_t mode, uint32_t operation,
```

Comment:
Would it make sense to have separate functions that call this function width different settings for the `bool`s? That way which operation mode is set is clear from reading test cases.

Reply:
I've used the standard values here to reduce the number of lines changed in order to facilitate the code review, but I can change it to make every call to this function explicit and make it clear to whom is reading the code.

Reply:
Prefer to optimize for readability in the future, especially when it comes to tests.